### DISCUSSION TITLE
Page display optimization

### DISCUSSION THREAD

**Discussion by zhangshenghang** [apache/hertzbeat#1951] [General]
When there are too many monitoring indicators, the current chart cannot display it better. Is there any plan to optimize the page in the future? For example, charts can be customized in size and position by dragging and dropping.
Or have other better plans?
<img width="642" alt="image" src="https://github.com/apache/hertzbeat/assets/29418975/bc620703-fea2-42c8-bbfc-247af0de1937">

**Comment by tomsun28** [comment]
👍 Hi, because we automatically generate these charts based on the defined yml, it is recommended that we generate them automatically later to generate better-looking adaptive charts, such as automatically expanding the table width.
For custom chart requirements, integration of grafana features is already being developed, see https://github.com/apache/hertzbeat/pull/1658
